Documentation/deploy-master-single.md

```diff
@@ -112,7 +112,7 @@ spec:
     - --service-cluster-ip-range=${SERVICE_IP_RANGE}
     - --secure_port=443
     - --advertise-address=${ADVERTISE_IP}
-    - --admission-control=NamespaceLifecycle,NamespaceExists,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
+    - --admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
This one is mainly a bootstrapping problem. The kube-system namespace does not exist by default, unless something has changed. So we have a decision to make. If you bootstrap the api server using a pod under the kubelet in the kube-system namespace, then you need to handle the race. I think this issue came up because we are trying to deploy the api server in the kube-system namespace before it exists.